### PR TITLE
57 navigation bar

### PR DESCRIPTION
**Pull request by alaa-yasin**

### CODE REVIEW

File: src/components/navigationBar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <div className="navbar">
+      <nav>
+        <i>
+          <HomeIcon />
+        </i>
+        <i>
+          <Calendar />
+        </i>
+        <i>
+          <AddNew className="addnewicon" />
```
Comment (yosefanajjar) [inline src/components/navigationBar/index.jsx]: Use BEM, as you can call it -> navbar__icon

**Review by fares98 — DISMISSED** [review]

File: src/components/navigationBar/navigation.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.navbar {
```
Comment (AlaaTaima) [inline src/components/navigationBar/navigation.css:1]: try to use flex , it's better than position

File: src/components/navigationBar/index.jsx (hunk 1 above)
Comment (alaa-yasin) [reply src/components/navigationBar/index.jsx]: okay

**Review by fares98 — DISMISSED** [review]

**Comment by Fatmasiam** [comment]
Write description

**Review by fares98 — DISMISSED** [review]

File: src/components/navigationBar/navigation.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+@media all and (min-width: 768px) and (max-width: 1024px) {
+  nav i + i {
+    padding-left: 5em;
+  }
+}
+
+@media all and (min-width: 1025px) and (max-width: 2560px) {
+  nav i + i {
```
Comment (yosefanajjar) [inline src/components/navigationBar/navigation.css]: use classes name

**Review by Fatmasiam — DISMISSED** [review]

**Review by yosefanajjar — DISMISSED** [review]

**Review by AlaaTaima — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Review by yosefanajjar — APPROVED** [review]

**Review by fares98 — APPROVED** [review]

**Pull request merged**